common/src/main/resources/docker/docker.bom

```diff
@@ -433,6 +433,10 @@ brooklyn.catalog:
 
         launch.command: |
           docker run -d ${IMAGE_DETAILS}
+          
+        checkRunning.command: |  
+          result="$(docker ps | grep ${IMAGE_DETAILS} | wc -l)"
+          if [ $result -ne 0 ]; then exit 0; else exit 1; fi;
```

Comment:
good point, but I wanted to check that the specific image was running.
Do you think it is ok to merge without?